examples/validation/validate_line_magnetics.py

```diff
@@ -0,0 +1,152 @@
+"""

```

Comment:
I would put this file to tests-folder and the script to a test function so that we can use pytest for it

Reply:
Overall this is very good, thanks for the fix!

Reply:
>could you allow me to push to your fork. There should be a box like "Allow edits from maintainers" or similar that you can tick.
Hm, I think this box was already ticked. Do you still have an access problem?

![image](https://user-images.githubusercontent.com/33814830/137022584-edb5a47f-308c-4915-b541-97a75035a2f5.png)

> I would put this file to tests-folder and the script to a test function so that we can use pytest for it
Makes sense.

I was happy to help. Your package was quite useful to me at some point.

Reply:
Good, my mistake in pushing. Now I managed to update this.